plugins/rikki/heroeslounge/http/Match.php

```diff
@@ -78,25 +83,40 @@ public function games($id)
         return $retVal;
     }
 
-    public function getTodaysMatches($date, $tz1 = TimezoneHelper::DEFAULT_TIMEZONE, $tz2 = "")
+    private function formatTimezone($tz1, $tz2)
     {
         if ($tz2) {
-            $timezone = $tz1 . '/' . $tz2;
+            return $tz1 . '/' . $tz2;
         } else {
-            $timezone = $tz1;
+            return $tz1;
         }
+    }
 
-        // get today in a given timezone
-        if ($date == 'today') {
-            $start = Carbon::today($timezone);
+    private function getMatchesForDate($start, $isPlayed)
+    {
+        // Convert to default/server timezone for db query
+        $start->setTime(0,0,0)->setTimezone(TimezoneHelper::DEFAULT_TIMEZONE);
+        $end = $start->copy()->addDay();
+
+        if (!$isPlayed) {
+            return MatchModel::whereBetween('wbp', [$start, $end])->where('is_played',false)->get();
```

Comment:
This should probably use $isPlayed

Reply:
Do you ever call `isPlayed` as true anywhere?